src/picongpu/include/plugins/adios/ADIOSWriter.hpp

```diff
@@ -421,10 +421,15 @@ class ADIOSWriter : public ILightweightPlugin
              (&mThreadParams.adiosAggregators)->default_value(0), "Number of aggregators [0 == number of MPI processes]")
             ("adios.ost", po::value<uint32_t > (&mThreadParams.adiosOST)->default_value(1),
              "Number of OST")
+            ("adios.create-meta", po::value<uint32_t > (&mThreadParams.adiosCreateMeta)->default_value(1),
```

Comment:
Please use something like this `("adios.disable-meta", po::value<bool>(&mThreadParams.adiosDisableMeta)->default_value(false), "Gather and write a global meta file, can be time consuming (use 0 with `bpmeta` post-mortem).");`